dockerfiles/99-fomu.rules

```diff
@@ -0,0 +1 @@
+SUBSYSTEM=="usb", ATTRS{idVendor}=="1209", ATTRS{idProduct}=="5bf0", MODE="0664", GROUP="plugdev"
```

Comment:
Since this is the only Dockerfile for now, and the rules apply not only to the container, I would move both sources to the root of the repo.